{
  "uid" : "1672720c89e8d90c",
  "name" : "starts empty",
  "fullName" : "Contract: EnumerableMap.starts empty",
  "historyId" : "c6f05c2efec1aef65d190b59c0a2423c",
  "time" : {
    "start" : 1635782774718,
    "stop" : 1635782775673,
    "duration" : 955
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0420a90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0420a90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/utils/structs/EnumerableMap.test.js:16:43)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: EnumerableMap"
  }, {
    "name" : "testClass",
    "value" : "Contract: EnumerableMap"
  }, {
    "name" : "testMethod",
    "value" : "starts empty"
  }, {
    "name" : "package",
    "value" : "Contract: EnumerableMap"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "1f6ef379001043a7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a06837c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782710360,
        "stop" : 1635782710922,
        "duration" : 562
      }
    }, {
      "uid" : "1af0e3ed33f4efb4",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0648250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782671626,
        "stop" : 1635782672190,
        "duration" : 564
      }
    }, {
      "uid" : "22798bc417564abc",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0689070>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782660605,
        "stop" : 1635782661175,
        "duration" : 570
      }
    }, {
      "uid" : "8d8819afaf2570ac",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2875190>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782639695,
        "stop" : 1635782640821,
        "duration" : 1126
      }
    }, {
      "uid" : "be006b415257157a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04374c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782619419,
        "stop" : 1635782619980,
        "duration" : 561
      }
    }, {
      "uid" : "5bd511f26ab89e44",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0428130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782554747,
        "stop" : 1635782555689,
        "duration" : 942
      }
    }, {
      "uid" : "d0e4ab08560eafb9",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7a160>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782554718,
        "stop" : 1635782555283,
        "duration" : 565
      }
    }, {
      "uid" : "2485348512794e30",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0680850>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782552951,
        "stop" : 1635782553514,
        "duration" : 563
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "b205ec98f1386ac2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/b205ec98f1386ac2",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590812595,
          "stop" : 1635590824448,
          "duration" : 11853
        }
      }, {
        "uid" : "6d9dd143a83ed98e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/6d9dd143a83ed98e",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c3640>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555054447,
          "stop" : 1635555055449,
          "duration" : 1002
        }
      }, {
        "uid" : "2e4d8c1a63c55c03",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/2e4d8c1a63c55c03",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551258767,
          "stop" : 1635551258837,
          "duration" : 70
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1672720c89e8d90c.json",
  "parameterValues" : [ ]
}